plugins/security/nexus-ldap-plugin/src/test/java/org/sonatype/security/realms/ldap/internal/connector/dao/LdapGroupDAOIT.java

```diff
@@ -46,10 +47,10 @@ public void testUsingDNInGroupMemberFormat()
   protected void doTestWithGroupMemberFormat(String groupMemberFormat)
       throws Exception
   {
-    Map<String, Object> env = new HashMap<String, Object>();
+    Map<String, Object> env = Maps.newHashMap();
     // Create a new context pointing to the overseas partition
     env.put(Context.INITIAL_CONTEXT_FACTORY, "com.sun.jndi.ldap.LdapCtxFactory");
-    env.put(Context.PROVIDER_URL, "ldap://localhost:" + getLdapServer().getPort() + "/o=sonatype");
+    env.put(Context.PROVIDER_URL, "ldap://localhost:" + ldapServers.get("default").getPort() + "/o=sonatype");
```

Comment:
might even be worthwhile pulling InitialLdapContext config to LdapITSupport, duplication now extends across its child classes